src/ontology/mondo-edit.obo

```diff
@@ -87487,32 +87464,48 @@ synonym: "acute T-cell lymphoblastic leukaemia" EXACT OMO:0003005 []
 synonym: "acute T-cell lymphoblastic leukemia" EXACT [NCIT:C3183]
 synonym: "acute T-cell lymphocytic leukaemia" EXACT OMO:0003005 []
 synonym: "acute T-cell lymphocytic leukemia" EXACT [NCIT:C3183]
+synonym: "precursor T lymphoblastic leukaemia" EXACT OMO:0003005 []
+synonym: "precursor T lymphoblastic leukemia" EXACT [DOID:5603, NCIT:C3183]
 synonym: "precursor T-lymphoblastic leukaemia" EXACT OMO:0003005 []
 synonym: "precursor T-lymphoblastic leukaemia (T-cell ALL)" EXACT OMO:0003005 []
 synonym: "precursor T-lymphoblastic leukemia" EXACT [NCIT:C3183]
 synonym: "precursor T-lymphoblastic leukemia (T-cell ALL)" EXACT [NCIT:C3183]
 synonym: "T acute lymphoblastic leukaemia" EXACT OMO:0003005 []
 synonym: "T acute lymphoblastic leukemia" EXACT [NCIT:C3183]
 synonym: "T-ALL" EXACT [NCIT:C3183]
-synonym: "T-cell acute lymphoblastic leukemia" EXACT [NCIT:C3183]
+synonym: "T-cell acute lymphoblastic leukaemia" RELATED OMO:0003005 []
```

Comment:
this should be an exact synonym (since the same synonym - in American English - is exact)